src/fragments/lib/in-app-messaging/integrate-your-application/integrate-your-application.mdx

```diff
@@ -0,0 +1,59 @@
+In your application directory, you should first install the necessary dependencies for using In-App Messaging.
+
+import reactnative0 from '/src/fragments/lib/in-app-messaging/integrate-your-application/react-native/install-dependencies.mdx';
+
+<Fragments fragments={{ 'react-native': reactnative0 }} />
+
+import js0 from '/src/fragments/lib/in-app-messaging/integrate-your-application/js/install-dependencies.mdx';
+
+<Fragments fragments={{ js: js0 }} />
+
+### Configure Amplify
```

Comment:
For React, the Amplify configuration code doesn't appear in the compete code sample below.

Reply:
Our recommended approach has been to configure Amplify before getting into the React application itself - which is typically in a separate entry file.